### PR TITLE
[#110] Add support for Ora

### PR DESCRIPTION
**Pull request by pmeinhardt**
Add support for Ora. More specifically, allow Tickety-Tick to pick up information from opened tasks (information is extracted for single tasks that are opened in a model when clicking on them).

Addresses #110.

### CODE REVIEW

File: spec/adapters/ora-spec.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  `;
+}
+
+describe('ora adapter', () => {
+  const loc = { host: 'ora.pm', pathname: '/project/53123/kanban/task/641617' };
```
Comment (klappradla) [inline spec/adapters/ora-spec.js]: Does it also work in the _list_ view: `/project/53123/list/task/641617` ?

**Comment by klappradla** [comment]
Will test-drive it immediately 🚀

File: spec/adapters/ora-spec.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        </form>
+      </div>
+      <div class="under-title-container">
+        <div data-ng-hide="::!editRights" class="submenu">
+          <div data-ng-if="features.visible_id" class="task-id">
+            <span>#${id}</span>
+          </div>
+          <div data-ng-if="features['task_types']" data-is-open="taskTypesPopover.open">
+            <div class="text dropdown-toggle" aria-expanded="false">
+              <span class="hide-mobile capitalize-first-letter truncate70">${type}</span>
```
[the remark this answers is not in the document]
Comment (pmeinhardt) [reply spec/adapters/ora-spec.js]: Yeah 🤷‍♂️ you never know 😀 However this is done in CSS so it won't affect the markup and our Tickety-Tick code which is nice.

File: spec/adapters/ora-spec.js (hunk 1 above)
Comment (pmeinhardt) [reply spec/adapters/ora-spec.js]: I have only started working with Ora so I wasn't even aware there's a list view. So, no, the current version is just a quick first stab at supporting Ora and works only for single, opened tasks.

File: spec/adapters/ora-spec.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (klappradla) [reply spec/adapters/ora-spec.js]: Oh, cool! With all the AngularJS things jumping around there I didn't even realize it's _"just"_ a class 🙈

**Comment by klappradla** [comment]
🤔 somehow `yarn install` took +2 minutes for me 🤔

**Comment by klappradla** [comment]
Yay, works on view as well ✅

**Comment by pmeinhardt** [comment]
Here's a screenshot @klappradla…

image

**Comment by klappradla** [comment]
Should we ⛵️ ? Or do you want to have another look @nickgnd ?

**Comment by nickgnd** [comment]
@klappradla looks awesome, please ship it 🚀

**Comment by klappradla** [comment]
FYI @pmeinhardt: I'll quickly draft a release.

**Pull request merged**